scripts/rest_examples/reservation-create-gcp.http

```diff
@@ -4,12 +4,12 @@ Content-Type: application/json
 X-Rh-Identity: {{identity}}
 
 {
-  "name": "gcp-linux-us-east4-a",
-  "zone": "us-east4-a",
+  "name": "gcp-linux-eu-west8-c",
+  "zone": "europe-west8-c",
```

Comment:
east4-a was full. Since GCP has global images IIUC and most of the team is in EU zone, I think something closer might make sense.